modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/utilities/index.js

```diff
@@ -16,6 +16,23 @@ import {fetch} from 'frontend-js-web';
 
 import createOdataFilter from './odata';
 
+export function getAcceptLanguageHeaderParam() {
+	const browserLang = navigator.language || navigator.userLanguage;
+	const themeLang = Liferay.ThemeDisplay.getLanguageId().replace('_', '-');
+
+	if (browserLang === themeLang) {
+		return browserLang;
+	}
+
+	return `${browserLang}, ${themeLang};q=0.8`;
+}
+
+const fetchHeaders = new Headers({
```

Comment:
The `fetch` of `frontend-js-web` already deals with creating a Header, so we just need to pass an object and it will take care of the rest.